### PR TITLE
Add MaxTxGasWanted to filter out large gas txs

### PR DESCRIPTION
**Pull request by yzang2019**

### CODE REVIEW

File: internal/mempool/mempool.go [hunk 1]
```diff
@@ -368,6 +368,10 @@ func (txmp *TxMempool) CheckTx(
 	return nil
 }
 
+func (txmp *TxMempool) shouldExclude(wtx *WrappedTx) bool {
+	return txmp.config.MaxTxGasWanted > 0 && wtx.gasWanted > txmp.config.MaxTxsBytes
```
Comment (stevenlanders) [inline internal/mempool/mempool.go:372]: > `MaxTxGasWanted` ?

Comment (stevenlanders) [inline internal/mempool/mempool.go:372]:
```suggestion
	return txmp.config.MaxTxGasWanted > 0 && wtx.gasWanted > txmp.config.MaxTxGasWanted
```

Comment (yzang2019) [reply internal/mempool/mempool.go:372]: Fixed

Comment (yzang2019) [reply internal/mempool/mempool.go]: Fixed

**Review by stevenlanders — APPROVED** [review]

**Pull request closed without merging**